libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,21 @@
+<nav role="navigation" attr.aria-label="Pagination Navigation">
+  <a attr.aria-label="Go to Previous Page" tabindex="0" (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev">
+    <fa-icon [icon]="['fa', 'chevron-left']"></fa-icon>Prev
+  </a>
+
+  <span class="daff-paginator__ellipsis" *ngIf="showFirstEllipsis()"><a tabindex="0" attr.aria-label="Go to Page 1" class="daff-paginator__page-link" (click)="onNotifyPageChange(1)" [class.daff-paginator__page-link--selected]="isSelected(1)">1</a>...</span>
+  <span *ngFor="let pageNumber of _numberOfPagesArray">
+    <a tabindex="0" attr.aria-label="Go to Page {{pageNumber + 1}}"
+      aria-current="isSelected(pageNumber + 1)"
+      class="daff-paginator__page-link" 
+      [class.daff-paginator__page-link--selected]="isSelected(pageNumber + 1)" 
+      *ngIf="showNumber(pageNumber + 1)"
```

Comment:
Could this conditional be built into the calculation of `_numberOfPagesArray` instead?

Reply:
I don't think so, because the `*ngFor` iterates over an array and doesn't care what's inside. So we'd need to make `*ngFor` base 1 instead of base 0.

Reply:
Oh, wait. Actually maybe this can be done

Reply:
done